### PR TITLE
feat(personio): support new booking files format

### PR DESCRIPTION
**Pull request by zyv**
Personio has just released new booking file format, adjust code to support both.

### CODE REVIEW

**Review by catcombo — APPROVED** [review]
Didn't test it, but it looks good. Despite using lists instead of tuples ;)

File: sepacetamol/views/datev.py [hunk 1]
```diff
@@ -204,10 +204,26 @@ def model_dump_csv_header(self) -> str:
...
+        datetime.strptime(datum, "%d.%m.%Y").replace(tzinfo=ZoneInfo("Europe/Berlin")).date()
+        if isinstance(datum, str)
+        else datum.date()
+    )
+
+    if isinstance(belegfeld_1, float):
+        belegfeld_1 = str(int(belegfeld_1))
 
-    datum = datetime.strptime(datum, "%d.%m.%Y").replace(tzinfo=ZoneInfo("Europe/Berlin")).date()
+    buchungstext = (buchungstext[:57] + "..") if len(buchungstext) > 60 else buchungstext
```
Comment (mm-matthias) [inline sepacetamol/views/datev.py:226]:
```suggestion
    buchungstext = (buchungstext[:57] + "...") if len(buchungstext) > 60 else buchungstext
```

**Review by mm-matthias — APPROVED** [review]

File: sepacetamol/views/datev.py (hunk 1 above)
Comment (zyv) [reply sepacetamol/views/datev.py]: Ha-ha, good catch, thank you!

**Pull request merged**